README.yaml

```diff
@@ -0,0 +1,67 @@
+name: "grafana-dashboards"
+# Canonical GitHub repo
+github_repo: "cloudposse/grafana-dashboards"
+# License of this project
+license: "APACHE2"
+# Badges to display
+badges:
+  - name: "Latest Release"
+    image: "https://img.shields.io/github/release/cloudposse/grafana-dashboards.svg"
+    url: "https://github.com/cloudposse/grafana-dashboards/releases"
+  - name: "Slack Community"
+    image: "https://slack.cloudposse.com/badge.svg"
+    url: "https://slack.cloudposse.com"
+
+description: |-
+  Grafana dashboards imported from open source projects and proccessed to be imported with grafana dashboard provisioner.
+
+introduction: |-
+  Starting from [Grafana 5.0](http://docs.grafana.org/administration/provisioning/#dashboards) there is provisioning
+  system that uses config files to create\update dashboards.
+
+  [The problem](https://github.com/grafana/grafana/issues/10786) with the provisioner is that it does not support
+  dashboards variables.
+
+  This project is about workaround for the problem.
+  It fetch dashboard files and process them replacing `${DS_PROMETHEUS}` with `Prometheus` value and save it.
```

Comment:
It fetches dashboard files and processes them replacing `${DS_PROMETHEUS}` with `Prometheus`, and then saves it